go.mod

```diff
@@ -162,13 +162,14 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 	istio.io/api v0.0.0-20190820204432-483f2547d882
 	istio.io/gogo-genproto v0.0.0-20190819131816-7a8328e41c1a
-	istio.io/operator v0.0.0-20190830172131-647a5416137b
+	istio.io/operator v0.0.0-20190918235823-f754c5665bbf
 	istio.io/pkg v0.0.0-20190731230704-fcbac27d69d5
 	k8s.io/api v0.0.0-20190222213804-5cb15d344471
 	k8s.io/apiextensions-apiserver v0.0.0-20190221221350-bfb440be4b87
 	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
 	k8s.io/cli-runtime v0.0.0-20190221101700-11047e25a94a
 	k8s.io/client-go v10.0.0+incompatible
 	k8s.io/helm v2.13.1+incompatible
+	k8s.io/kubernetes v1.13.1
```

Comment:
I don't think depending on k8s.io/kubernetes is a good idea/recommended. not strictly related to this pr though